docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,11 @@
+# Only allow admins to create new streams
+By enabling this will only allow administrators to create new streams within
+the organisation. This can only be configured from an Administrator's account.
+1. Locate the cog
```

Comment:
'1.' should be on a new line:
![screen shot 2016-12-27 at 11 34 16 am](https://cloud.githubusercontent.com/assets/641763/21492667/79362526-cc28-11e6-835f-7c11c998779f.png)